app/models/guest_application.rb

```diff
@@ -1,6 +1,7 @@
 class GuestApplication < ActiveRecord::Base
   belongs_to :vm_or_template
   belongs_to :host
+  belongs_to :container_image
```

Comment:
@abonas there are actually already other **two** `belongs_to`, one for `:vm_or_template` and one for `:host` (which are logically exclusive). This is a strategy used in several tables in ManageIQ (e.g. EventStream, etc.). Not sure if it's optimal (cc @blomquisg) but it's out of scope here refactoring the GuestApplication table.